### DISCUSSION TITLE
Does it support 32 bit?

### DISCUSSION THREAD

**Discussion by hgup** [neovide/neovide#1101] [Q&A]
Someone asked me this and I was clueless.
I don't have a 32 bit Windows operating system to test it out. So, can you confirm this?

Also, can we build it on 32 bit systems?

**Comment by Kethku** [comment]
TBH I haven't ever tried. I have no idea if it would work or not, but it sorta fits under likely not supported.

Neovide uses relatively recent features of the gpu through skia and assumes a pretty modern machine. It wasn't intentionally built not to work on such systems, but isn't tested.
